tests/test_hedgehog_smoke.py

```diff
@@ -326,6 +326,182 @@ def test_pmon_smoke(setup):
+
+    # Check CLI available
+    run_shell_helper(duthost, shell_show_nat, "CLI \"{}\" command error".format(shell_show_nat), do_assert=True)
+    run_shell_helper(duthost, shell_conf_nat, "CLI \"{}\" command error".format(shell_conf_nat), do_assert=True)
+
+    # Get NAT state
+    res_show_nat_state = run_shell_helper(duthost, shell_show_nat_state, "Fail while getting NAT state", do_assert=True)
+    original_nat_state = res_show_nat_state['stdout']
+    new_nat_state = "enable" if original_nat_state == "disabled" else "disable"
+
+    # Change NAT state and check it
+    run_shell_helper(duthost, shell_conf_nat_feature + " " + new_nat_state)
+    res_show_nat_state = run_shell_helper(duthost, shell_show_nat_state, "CLI \"{}\" command error".format(shell_show_nat_state), do_assert=True)
+    pytest_assert(original_nat_state != res_show_nat_state['stdout'], "Error while changing NAT feature state")
+
+    # cleanup
+    run_shell_helper(duthost, shell_conf_nat_feature + " " + original_nat_state)
+
+
+def test_radius_smoke(setup):
+    duthost = setup['duthost']
+    config = setup['config']
+    build_flag = "INCLUDE_RADIUS"
+    deb_packets = ["libnss-radius", "libpam-radius-auth"]
+    shell_show_radius = "show radius"
+    shell_show_radius_server = shell_show_radius + " | grep \"RADIUS_SERVER\" | awk '{print $3}'"
+    fake_server_ip = "1.1.1.1"
+    shell_conf_radius = "sudo config radius"
+    shell_conf_radius_add = shell_conf_radius + " add " + fake_server_ip
+    shell_conf_radius_del = shell_conf_radius + " del " + fake_server_ip
+
+    if config and config[build_flag] == "n":
+        pytest.skip("SKIP. Radius feature is disabled on build.")
+
+    # Check if deb packets installed
+    for package in deb_packets:
+        shell_check_package = "apt list --installed " + package
+        res_check_package = run_shell_helper(duthost, shell_check_package)
+        pytest_assert(len(res_check_package['stdout_lines']) > 1, "No installed {} package".format(package))
+
+    # Check CLI available
+    run_shell_helper(duthost, shell_show_radius, "CLI \"{}\" command error".format(shell_show_radius), do_assert=True)
+    run_shell_helper(duthost, shell_conf_radius, "CLI \"{}\" command error".format(shell_conf_radius), do_assert=True)
+
+    # Check there is no radius server with fake IP
+    res_show_radius_server = run_shell_helper(duthost, shell_show_radius_server)
+    for line in res_show_radius_server['stdout_lines']:
+        pytest_assert(line != fake_server_ip, "Radius server with IP {} already exists")
+    
+    # Add fake Radius server
+    run_shell_helper(duthost, shell_conf_radius_add)
+    res_show_radius_server = run_shell_helper(duthost, shell_show_radius_server)
+    pytest_assert(fake_server_ip in res_show_radius_server['stdout_lines'], "Radius server {} doesn't added".format(fake_server_ip))
+
+    # Cleanup
+    run_shell_helper(duthost, shell_conf_radius_del)
+
+
+def test_ntp_smoke(setup):
```

Comment:
hmm, maybee add/delete fake ntp server (as was done for radius)?

Reply:
ntp returns "unsynchronised" error